CMake: Support finding system libwebp using pkg-config

cjmayo opened the pull request:
Installing libwebp using autotools is still supported.

---

The autotools libwebp build allows control over individual CPU flags, whereas the CMake one has a SIMD option which then automatically selects the CPU flags supported on the build machine.

cjmayo committed:
CMake: Support finding system libwebp using pkg-config

Installing libwebp using autotools is still supported.

vadz commented on build/cmake/modules/FindWebP.cmake:
```diff
@@ -0,0 +1,8 @@
+find_package(WebP CONFIG)
+
+if(NOT WebP_FOUND)
+    find_package(PkgConfig)
```
We do it like this elsewhere AFAICS:
```suggestion
    find_package(PkgConfig QUIET)
```

vadz reviewed (commented):
Maybe it's just me, but I find it confusing to use `find_package(WebP)` but then actually using `pkg_check_modules()` for it under the hood. It would, IMHO, be more clear to do it directly in `build/cmake/lib/webp.cmake`.

cjmayo committed:
CMake: Use QUIET with find_package(PkgConfig) for libwebp

cjmayo committed:
CMake: Remove find_package(WebP) from webp.cmake

find_package(WebP) has already been called because options.cmake
uses wx_add_thirdparty_library().
